Source/Core/Core/Core.vcxproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I figured out that this weird change is in fact *removing* the byte order mark that currently exists in the repo.